### PR TITLE
feat(server): add new /orders endpoints

### PR DESCRIPTION
**Pull request by nasjuice**
#

## Changes this PR introduces

- Added create/edit/delete/get endpoints for Order Page

## How to test this PR
Postman
-

## Notes for reviewers

-

## Checklist

- [ ] Have I renamed the title of this pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)?
- [ ] I will squash and merge when my PR is approved.
- [ ] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-n31rda Ordering page](https://app.clickup.com/t/n31rda)

**Comment by gordonpn** [comment]
Zeet is failing because there is a `-` character before `.zeet.app`

```
-ordering-.zeet.app/
         ^
```

File: server/app/models/OrderModel.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export interface IOrder extends Document {
+  componentName: string;
+  option1: string;
+  option2: string;
+  quantity: number;
+  cost: number;
+  deliveryDate: Date;
+  manufactureName: string;
+  note: string;
+}
```
Comment (gordonpn) [inline server/app/models/OrderModel.ts:12]: Should we store `orderDate` as well?

Wasn't the ordering page about ordering materials? And the production page was about parts and bikes.

Comment (nasjuice) [reply server/app/models/OrderModel.ts:12]: uhhh idk i based it off the ligma. but yessir order date should be on there.

**Pull request closed without merging**